docs/quickstart/run_model_on_rtsp.md

```diff
@@ -0,0 +1,63 @@
+You can run computer vision models on RTSP stream frames with Inference.
```

Comment:
This guide is nearly identical to the webcam guide. Should we combine them into a single guide?